awx/main/tasks.py

```diff
@@ -422,23 +442,29 @@ def discover_receptor_nodes():
         (changed, instance) = Instance.objects.register(hostname=hostname, node_type='execution')
         was_lost = instance.is_lost(ref_time=nowtime)
         if changed:
-            logger.info("Registered execution node '{}'".format(hostname))
+            logger.warn("Registered execution node '{}'".format(hostname))
             check_heartbeat.apply_async([hostname])
         else:
```

Comment:
I think this shouldn't be an `else:`. If it's a new execution node, then we should still set `modified` to the last-seen date as viewed by receptor.